### PR TITLE
removed Plots and UMAP, removed Manifest

### PR DESCRIPTION
**Pull request by mkarikom**
Just fixed previous PR as discussed

### CODE REVIEW

**Comment by slycoder** [comment]
Cool, just FYI you didn't need to close the previous PR (Linking here as #10  for reference).

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 .DS_Store
+*.jld
+Manifest.toml
```
Comment (slycoder) [inline .gitignore:3]: Should manifest be ignored or committed? If it's like package management in a few other languages it seems like something that should be kept to ensure that git-shas are maintained.

**Comment by mkarikom** [comment]
I think the general consensus is to not commit Manifest.toml (I just did a
quick check on DifferentialEquations.jl, Distributions.jl, Optim.jl),
although it is useful for for reproducing a specific environment.

For instance, if you had an MWE on an issue you could run "Pkg.activate
<cloned repo's root directory> and running Pkg.instantiate to pull in stuff
from "<cloned repo's root directory>/Manifest.toml".

However, Manifest.toml doesn't enforce versions of deps.  In fact, if you
call Pkg.build with no Manifest.toml present, it installs all deps at
versions consistent with the [compat] field in Project.toml, then generate
a Manifest.toml consistent with this, which by default is in the home
project (~/.julia/environments/v1.4/Manifest.toml).

I do generate a Manifest.toml in the package dir for my development
workflow but it only serves as a record.  It gets generated automatically
whenever I run Pkg.build with the project directory activated.

Hope that helps!

**Comment by slycoder** [comment]
Thank you for that explanation!

**Pull request merged**